packages/dx-react-chart-demos/src/demo-sources/chart-basic/chart-connect-props.jsxt

```diff
@@ -0,0 +1,146 @@
+import * as React from 'react';<%&additionalImports%>
+import { connectProps } from '@devexpress/dx-react-core';
+import {
+  Scale, EventTracker, SelectionState,
+} from '@devexpress/dx-react-chart';
+import {
+  Chart,
+  SplineSeries,
+  ArgumentAxis,
+  ValueAxis,
+  Legend,
+} from '@devexpress/dx-react-chart-<%&themeName%>';
+import { energyConsumption as data } from '../../../demo-data/data-vizualization';
+
+const rootStyles = {
+  display: 'flex',
+  margin: 'auto',
+  flexDirection: 'row',
+};
+const LegendRoot = ({ classes, ...restProps }) => (
+  <Legend.Root
+    {...restProps}
+    style={rootStyles}
+  />
+);
+
+const defaultLabelStyles = {
+  marginBottom: '8px',
+  whiteSpace: 'nowrap',
+  fontSize: '20px',
+  color: 'gray',
+};
+const selectedLabelStyles = {
+  ...defaultLabelStyles,
+  fontWeight: 'bold',
+  color: 'black',
+};
+const LegendLabel = ({
+  classes, selectedSeriesName, text, ...restProps
+}) => (
+  <div
+    {...restProps}
+    style={selectedSeriesName === text ? selectedLabelStyles : defaultLabelStyles}
+  >
+    {text}
+  </div>
+);
+
+const itemStyles = {
+  flexDirection: 'column-reverse',
+};
+const LegendItem = ({ classes, ...restProps }) => (
```

Comment:
Why do you skip the `classes` prop? Does it affect UI?

Reply:
Yes, this causes errors.